Mathlib/RingTheory/Algebraic/Integral.lean

```diff
@@ -90,41 +89,20 @@ variable (K L : Type*) {R S A : Type*}
 
 section Ring
 
-section Field
-
-variable [Field K] [Field L] [Ring A]
-variable [Algebra K L] [Algebra L A] [Algebra K A] [IsScalarTower K L A]
+variable [CommRing K] [Nontrivial K] [Ring A] [Algebra K A]
```

Comment:
Could you please rename `K` to `R` in a follow-up PR?
```suggestion
-- TODO: rename `K` to `R`
variable [CommRing K] [Nontrivial K] [Ring A] [Algebra K A]
```

Reply:
Changed in the last commit.